geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ConverterIngest.scala

```diff
@@ -98,27 +105,21 @@ class LocalIngestConverterImpl(sft: SimpleFeatureType, path: String, converters:
   override def close(): Unit = converters.returnObject(converter)
 }
 
-/**
- * Distributed job that uses converters to process input files
- *
- * @param sft simple feature type
- * @param converterConfig converter definition
- */
-class ConverterIngestJob(dsParams: Map[String, String],
-                         sft: SimpleFeatureType,
-                         converterConfig: Config,
-                         paths: Seq[String],
-                         libjarsFile: String,
-                         libjarsPaths: Iterator[() => Seq[File]])
-    extends AbstractIngestJob(dsParams, sft.getTypeName, paths, libjarsFile, libjarsPaths) {
+abstract class AbstractConverterIngestJob(dsParams: Map[String, String],
+                                          sft: SimpleFeatureType,
+                                          converterConfig: Config,
+                                          paths: Seq[String],
+                                          libjarsFile: String,
+                                          libjarsPaths: Iterator[() => Seq[File]])
+  extends AbstractIngestJob(dsParams, sft.getTypeName, paths, libjarsFile, libjarsPaths) {
 
   import ConverterInputFormat.{Counters => ConvertCounters}
   import GeoMesaOutputFormat.{Counters => OutCounters}
 
   val failCounters =
     Seq((ConvertCounters.Group, ConvertCounters.Failed), (OutCounters.Group, OutCounters.Failed))
 
-  override val inputFormatClass: Class[_ <: FileInputFormat[_, SimpleFeature]] = classOf[ConverterInputFormat]
+  def inputFormatClass: Class[_ <: FileInputFormat[_, SimpleFeature]]
```

Comment:
I think you can remove this line, as it's defined in the super class and there's no point in redefining it